### ISSUE TITLE
Error "text not allowed here" for a <note type="speaker"> 

### ISSUE THREAD

**Issue by nuriabel** [clarin-eric/ParlaMint#437]
Hi Tomaz, here the new issue, thanks for previous comment. 

We got an error for this code
ERROR
/mnt/d/UPF/proyecto_parlamint/ParlaMint/Data/ParlaMint-ES-CT/ParlaMint-ES-CT_2018-01-17-0101.xml:103:306: error: text not allowed here; expected element "gap", "incident", "kinesic", "note", "pb", "s" or "vocal"
/mnt/d/UPF/proyecto_parlamint/ParlaMint/Data/ParlaMint-ES-CT/ParlaMint-ES-CT_2018-01-17-0101.xml:103:312: error: element "seg" incomplete; expected element "gap", "incident", "kinesic", "note", "pb", "s" or "vocal"

103 is the <note type="speaker"> line. 

`<body>
         <div type="debateSection">
            <head>SESSIÓ NÚM. 1</head>
            <note>La sessió s'obre a les onze del matí i dos minuts. Presideix el president de la Mesa d’Edat, acompanyat dels secretaris de la Mesa d’Edat, la qual és assistida pel secretari general i el lletrat major.</note>
            <note>ORDRE DEL DIA DE LA CONVOCATÒRIA</note>
            <note>Punt únic: Constitució del Ple del Parlament i elecció de la Mesa del Parlament (tram. 396-00001/12 i 398-00001/12).</note>
            **<note type="speaker">El secretari general (Xavier Muro i Bas)</note>**
            <u xml:id="ParlaMint-ES-CT_2018-01-17-0101.1" who="MuroXavier" ana="member" xml:lang="ca">
               <seg xml:id="ParlaMint-ES-CT_2018-01-17-0101.1.0" xml:lang="ca">Bon dia a tothom, il·lustres senyores diputades i senyores diputats, autoritats que ens acompanyen, senyores i senyors, sigueu benvinguts al Parlament de Catalunya en aquesta sessió constitutiva de la seva dotzena legislatura.</seg>
              `
thanks!
N.

**Comment by TomazErjavec** [comment]
Much clearer, thanks. The problem is the "**" (twice), as you can't have text directly in the div element. A few other issues:
* the values of the pointing attributes here should start with "#"
* "member" is a not valid value for `u/@ana`, it should be either "#regular" or "#chair", cf. [Guidelines](https://clarin-eric.github.io/ParlaMint/#sec-uterrance)
* there is no need to put `@xml:lang="ca"` on utterances and segments - it should be on the top level TEI element, and is inherited from there.

So, in short (and assuming "member" is "regular"):
```
<u xml:id="ParlaMint-ES-CT_2018-01-17-0101.1" who="#MuroXavier" ana="#regular">
```

**Comment by nuriabel** [comment]
Thank you very much, now solved!

**Comment by rjzevallos** [comment]
Hello, I have same problem. For example my xml is

`<body>
   <div type="debateSection">
      <u xml:id="ParlaMint-ES-CT_2018-01-17-0101.1" who="#MuroXavier" ana="#regular">
         <seg xml:id="ParlaMint-ES-CT_2018-01-17-0101.1.0">Bon dia a tothom, il·lustres senyores diputades i senyores diputats 
         </seg>
         <note type="narrative">Constitució del Ple del Parlament i elecció de la Mesa del Parlament</note>
         <note type="narrative">396-00001/12 i 398-00001/12</note>
       </u>`

and I get this error:

error: text not allowed here; expected element "gap", "incident", "kinesic", "note", "pb", "s" or "vocal"
error: element "seg" incomplete; expected element "gap", "incident", "kinesic", "note", "pb", "s" or "vocal"

**Comment by TomazErjavec** [comment]
No idea what is wrong. The element content of `<u>` is
https://github.com/clarin-eric/ParlaMint/blob/47a6a842d5a6447266f3ce0d95ad83bdac66673e/Schema/ParlaMint-TEI.rng#L457-L462
but it is true that 
https://github.com/clarin-eric/ParlaMint/blob/47a6a842d5a6447266f3ce0d95ad83bdac66673e/Schema/ParlaMint-TEI.rng#L511 
is then quite complicated... Maybe I messed up the schema.
Can I get the whole file, i.e. is it on GitHub?

*But* why do you have "il·lustres" i.e. U+00B7 MIDDLE DOT? 
@matyaskopp, maybe we need an issue to discuss how far to perscribe [which characters to use](https://clarin-eric.github.io/ParlaMint/#sec-chars)?

**Comment by matyaskopp** [comment]
### validation
I don't see a problem with validation if you are speaking about this line in the file:
https://github.com/IULATERM-TRL-UPF/ParlaMint/blob/5016179013f12156b7782e6c54578b18109ba9cc/Data/ParlaMint-ES-CT/ParlaMint-ES-CT_2018-01-17-0101.xml#L103
```XML
<seg xml:id="ParlaMint-ES-CT_2018-01-17-0101.1.0" xml:lang="ca">Bon dia a tothom, il·lustres senyores diputades i senyores diputats, autoritats que ens acompanyen, senyores i senyors, sigueu benvinguts al Parlament de Catalunya en aquesta sessió constitutiva de la seva dotzena legislatura.</seg>
```
It passes the validation: https://github.com/clarin-eric/ParlaMint/actions/runs/3405493207/jobs/5663905394#step:4:1731
```
 INFO: Validating component TEI file /home/runner/work/ParlaMint/ParlaMint/ParlaMint/Data/ParlaMint-ES-CT/ParlaMint-ES-CT_2018-01-17-0101.xml
  INFO: XML validation for ParlaMint-ES-CT_2018-01-17-0101.xml
  INFO: Content validaton for ParlaMint-ES-CT_2018-01-17-0101.xml
  INFO: Link checking for ParlaMint-ES-CT_2018-01-17-0101.xml
```
### middle dot
The only strangeness is the middle dot, but the linguistic annotations process it:
```XML
<w xml:id="ParlaMint-ES-CT_2018-01-17-0101.1.0.1.6" ana="AQ0CP00" msd="UPosTag=ADJ" lemma="il·lustre">il·lustres</w>
```

My Wikipedia-based guess is that `l` and `MIDDLE DOT` should be one character: `ŀ` (U+0140)
@TomazErjavec, you can mention it in the documentation, but it is a language-specific issue (you don't want to say that `ž` shouldn't be written this way in Slovenian: `zˇ`). 
 
### lingv. annotation
Looking into your annotated version, it seems that you don't have UD features. The previous sample should be
```XML
<w xml:id="ParlaMint-ES-CT_2018-01-17-0101.1.0.1.6" ana="AQ0CP00" msd="UPosTag=ADJ|Number=Plur" lemma="il·lustre">il·lustres</w>
```
`Number=Plur` is missing.

**Comment by TomazErjavec** [comment]
> @TomazErjavec, you can mention it in the documentation, but it is a language-specific issue (you don't want to say that ž shouldn't be written this way in Slovenian: zˇ).

See #440.

**Comment by rjzevallos** [comment]
All my files are in https://github.com/IULATERM-TRL-UPF/ParlaMint/tree/data-ES-CT/Data/ParlaMint-ES-CT. 
On the other hand, the problem is not the characters because I have just changed and put another text (in Spanish), just to check, and the same error keeps coming up.

**Comment by matyaskopp** [comment]
It use ana schema:
```
  INFO: Validating component TEI.ana file /home/runner/work/ParlaMint/ParlaMint/ParlaMint/Data/ParlaMint-ES-CT/ParlaMint-ES-CT_2018-01-17-0101.xml
```
because you include it from `ParlaMint-ES-CT.ana.xml`
https://github.com/IULATERM-TRL-UPF/ParlaMint/blob/86731bc6d0fe4d135f398d23709614193409ec02/Data/ParlaMint-ES-CT/ParlaMint-ES-CT.ana.xml#L184-L185
```XML
<?xml version="1.0" encoding="UTF-8"?>
<teiCorpus xmlns="http://www.tei-c.org/ns/1.0"
           xml:id="ParlaMint-ES-CT.ana"
           xml:lang="ca">
...
   <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
               href="ParlaMint-ES-CT_2018-01-17-0101.xml"/>
</teiCorpus>
```

**Comment by TomazErjavec** [comment]
> All my files are in https://github.com/IULATERM-TRL-UPF/ParlaMint/tree/data-ES-CT/Data/ParlaMint-ES-CT

I have now cloned your repo and validated your component files., and I get
```
java -jar /usr/share/java/jing.jar /project/corpora/Parla/ParlaMint/ParlaMint/Schema/ParlaMint-TEI.rng \
ParlaMint-ES-CT/Data/ParlaMint-ES-CT/ParlaMint-ES-CT_2018-01-17-0101.xml \
ParlaMint-ES-CT/Data/ParlaMint-ES-CT/ParlaMint-ES-CT_2022-07-21-3602.

ParlaMint-ES-CT/Data/ParlaMint-ES-CT/ParlaMint-ES-CT_2022-07-21-3602.xml:100:16: 
error: element "div" incomplete; expected element "gap", "head", "incident", "kinesic", "note",
 "pb", "u" or "vocal"
```
so, the first file is ok, but the second one has:
```
      <body>
         <div type="debateSection">
            <head>Sessió 36, segona i darrera reunió, dijous 21 de juliol de 2022</head>
         </div>
          ...
```
The error here is that you cannot have a div element without a u, which, I think makes sense. 
However, this seems is a different fle and a different error from the one you reported - and I can't reproduce that one, even if I use the Schema/ParlaMint-TEI.rng from your repo. Hm.

**Comment by TomazErjavec** [comment]
Oh, sorry, only now read @matyaskopp's comment - he is, of course, right, you include a "plain text" file from the .ana root file, and this is then validated by the schema for linguistically annotated files, and it reports errors, as it should. You should have
```
<?xml version="1.0" encoding="UTF-8"?>
<teiCorpus xmlns="http://www.tei-c.org/ns/1.0"
           xml:id="ParlaMint-ES-CT.ana"
           xml:lang="ca">
...
   <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
               href="ParlaMint-ES-CT_2018-01-17-0101.ana.xml"/>
</teiCorpus>
```

**Comment by rjzevallos** [comment]
Yep, done. This problem was solved. There was an error with the url of the ana.xml file. Thank you.

**Comment by matyaskopp** [comment]
ok, closing
